src/sentry/reprocessing2.py

```diff
@@ -264,7 +264,68 @@ def reprocess_event(project_id, event_id, start_time):
     )
 
 
-def delete_old_primary_hash(event):
+def buffered_delete_old_primary_hash(event, force_flush_batch: bool = False):
+    """
+    Like `buffered_handle_remaining_events`, is a quick and dirty way to batch
+    event IDs so requests to tombstone rows are not being individually sent
+    over to Snuba.
+    """
+
+    from sentry import killswitches
+
+    if killswitches.killswitch_matches_context(
+        "reprocessing2.drop-delete-old-primary-hash", {"project_id": event.project_id}
+    ):
+        return
+
+    client = _get_sync_redis_client()
+    old_primary_hash = get_path(event.data, "contexts", "reprocessing", "original_primary_hash")
+
+    # TODO: revisit this comment
+    # We explicitly cluster by only project_id and old_primary_hash here such that our
+    # RENAME command later succeeds.
+    #
+    # We also use legacy string formatting here because new-style Python
+    # formatting is quite confusing when the output string is supposed to
+    # contain {}.
```

Comment:
this comment doesn't make a ton of sense (also not in the original code) because of an autoformatting rule that replaced `"re2:remaining:{%s:%s}` with this